README.md

```diff
@@ -4,92 +4,84 @@
 
-* `runClientTests` runs the client tests once.
-* `runClientTestsAndWatch` runs the client tests every time that the code changes after a save.
-* `runClientTestsWithCoverage` runs the client tests and deposits code coverage statistics into a new directory within `client` called `coverage`. In there you will find an `index.html`. Right click on `index.html` and select `Open in Browser` with your browser of choice. For Chrome users, you can drag and drop index.html onto chrome and it will open it.  
-* `runE2ETest` runs end to end test for the client side. What are e2e tests? They are tests that run the real application and simulate user behavior. They assert that the app is running as expected. NOTE: The server (`run`) needs to be on for this test to work!
+* `ng test` runs the client tests once.
+* `ng test --code-coverage` runs the client tests and generates a coverage file you can find in your client directory `client/coverage/client/index.html`.
+Right click on `index.html` and select `Copy path` and paste it into your browser of choice. For Chrome users, you can drag and drop `index.html` onto the tab area of Chrome and it will open it.  arma tests for the client, 
+* `ng e2e` runs end to end tests. arma tests for the client, What are e2e tests? They are tests that run the real aarma tests for the client, pplication and simulate user behavior. They assert that tharma tests for the client, e app is running as expected. NOTE: The server (`run`) needs to be actively executing for these tests to work!
 
-Turn on your repo in [Travis CI][travis], replace the build status image in this README, and push your changes. That will trigger a build with Travis.
+There are GitHub Actions set up in your repo for each of the three checks: JUnit tests for the server, Karma tests for the client, and Protractor tests for end-to-end testing. There are badges above that show the status of these checks on the master branch.
 
 ## Resources
 
+- [Angular tutorial][angular-tutorial]
+- [Angular testing (Karma)][angular-karma]
 - [Angular Tour of Heroes tutorial][tour-of-heroes]
+- [End-to-end testing (Protractor)][protractor]
+- [End to end testing (e2e) with protractor and Angular CLI][e2e-testing]
 - [What are environments in Angular CLI?][environments]
-- [Testing Angular 2 with Karma/Jasmine][angular4-karma-jasmine]
-- [End to end testing (e2e) with protactor and Angular CLI][e2e-testing]
 - [Angular CLI commands][angular-cli-commands]
 - [HTTP Status Codes][status-codes]
 
 
+[angular-tutorial]: https://angular.io/start
+[angular-karma]:https://angular.io/guide/testing
 [tour-of-heroes]: https://angular.io/tutorial
-[angular4-karma-jasmine]: https://codecraft.tv/courses/angular/unit-testing/jasmine-and-karma/
+[protractor]: https://www.protractortest.org/#/toc
 [e2e-testing]: https://coryrylan.com/blog/introduction-to-e2e-testing-with-the-angular-cli-and-protractor
 [environments]: http://tattoocoder.com/angular-cli-using-the-environment-option/
```

Comment:
This might be a better link for environments: https://angular.io/guide/build#configuring-application-environments